packages/create-plugin/templates/common/_package.json

```diff
@@ -26,8 +26,9 @@
     "@swc/core": "^1.3.90",
     "@swc/helpers": "^0.5.0",
     "@swc/jest": "^0.2.26",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^12.1.4",
+    "@testing-library/jest-dom": "6.1.4",
+    "@types/testing-library__jest-dom": "5.14.8",
```

Comment:
mega nit: move this to line 36 so package managers don't move it "randomly" when someone adds a package via cli.